src/applications/veteran-id-card/config.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
This content update required an unordered list. I think just changing this file to .jsx and importing React is all I need in order for this to support jsx partials.